packages/amplication-client/src/Components/ResourceCircleBadge.tsx

```diff
@@ -0,0 +1,34 @@
+import { CircleBadge, Icon } from "@amplication/design-system";
+import React from "react";
+
+interface Props {
+  type: "settings" | "service" | "api" | "queue";
+  size?: "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
```

Comment:
I think we should export the same type used in the circleBadge

Reply:
exactly, this is why we should not declare it twice. instead just export the inner type and use it here